images/linux/scripts/SoftwareReport/SoftwareReport.Tools.psm1

```diff
@@ -8,6 +8,13 @@ function Get-AnsibleVersion {
     return "Ansible $ansibleVersion"
 }
 
+function Get-AptFastVersion {
+    $result = Get-CommandResult "apt list --installed" -Multiline
```

Comment:
nitpick: `dpkg-query --showformat='${Version}' --show apt-fast`

Reply:
Thanks, it looks more clear!